README.md

```diff
@@ -210,6 +211,27 @@ For dex.decred.org, reach out
 [on Element](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org?via=decred.org&via=matrix.org&via=planetdecred.org)
 to appeal.
 
+## Fees
+
+The dex does not charge fees, but users have to pay on-chain transaction fees.
```

Comment:
Currently, there is a sign up fee. So I would rather this be "The dex does not charge **trading** fees."